ansible/playbook/tasks/roles/initialization/tasks/main.yml

```diff
@@ -1,5 +1,10 @@
 ---
 # This role contains tasks executed during initialization of PMM Server
+- name: Update repository settings
+  command: yum-config-manager --setopt=epel.timeout=1 --save
+  args:
+    warn: false
```

Comment:
Let's not suppress the messages :)

Reply:
I've enabled the warnings and set 'ignore_errors: True'.